src/test/java/uk/gov/hmcts/reform/professionalapi/controller/request/validator/UserCreationRequestValidatorTest.java

```diff
@@ -7,38 +7,24 @@
 
 import java.lang.reflect.Constructor;
 import java.util.List;
-
-import org.junit.Before;
 import org.junit.Test;
 import uk.gov.hmcts.reform.professionalapi.controller.request.InvalidRequest;
-import uk.gov.hmcts.reform.professionalapi.controller.request.validator.UserCreationRequestValidator;
-import uk.gov.hmcts.reform.professionalapi.domain.PrdEnum;
-import uk.gov.hmcts.reform.professionalapi.domain.PrdEnumId;
 
 public class UserCreationRequestValidatorTest {
 
-    private List<PrdEnum> prdEnumList;
-
     final String userManagerRole = "pui-user-manager";
     final String caseManagerRole = "pui-case-manager";
-    final String sidamRole = "SIDAM_ROLE";
 
-    @Before
-    public void setUp() {
-        prdEnumList = asList(
-                new PrdEnum(new PrdEnumId(2, sidamRole), userManagerRole, sidamRole),
-                new PrdEnum(new PrdEnumId(3, sidamRole), caseManagerRole, sidamRole));
-    }
 
     @Test
     public void test_validateRoles() {
-        List<String> validatedRoles = validateRoles(asList(userManagerRole, caseManagerRole), prdEnumList);
+        List<String> validatedRoles = validateRoles(asList(userManagerRole, caseManagerRole));
         assertFalse(validatedRoles.isEmpty());
     }
 
     @Test(expected = InvalidRequest.class)
     public void test_validateRolesThrows40WhenVerifiedRolesIsEmpty() {
```

Comment:
error code should be 400 in the name of the method?

Reply:
Done.